eng/build/Workers.Python.props

```diff
@@ -0,0 +1,15 @@
+<Project>
+
+  <ItemGroup>
+    <!-- Python worker does not ship with the host for windows. -->
+    <PackageReference Include="Microsoft.Azure.Functions.PythonWorker" Version="4.30.3" Condition="!$(RuntimeIdentifier.StartsWith('win'))" />
+  </ItemGroup>
+
+  <!-- PythonWorker package injects its content files through custom targets. -->
+  <Target Name="SuppressPythonWorker" AfterTargets="Initialize" BeforeTargets="CopyOnPublish" Condition="$(RuntimeIdentifier.StartsWith('win'))">
```

Comment:
@jviau I didn't completely understand this step? 
Is this step required because this [PR](https://github.com/Azure/azure-functions-python-worker/pull/1472) is not merged yet?

Reply:
Following up on this, it looks like the worker is only included if the runtime isn't windows. The step below removes the worker for windows, though. 

Shouldn't the worker already not be included if the runtime is windows? Why would the removal step be necessary?

Reply:
We have not been including the python worker for windows site extension for a while. I believe it is already on the machine (@fabiocav to clarify).

Today this is done as part of the script: https://github.com/Azure/azure-functions-host/blob/dev/build/build-extensions.ps1#L95. So it is removed after msbuild copies it. This approach avoids the copy altogether when on windows.

Note: this does not affect _local_ development, where `RuntimeIdentifier=""`, so python will be copied.

@gavin-aguiar its tangential to that PR you linked. That PR is to refactor python targets to make it play nicer with other MSBuild targets (avoid using such general names like `Initialize`, `SourceFiles`, as well as participate in an existing copy task)